modules/article-page/views/show.html

```diff
@@ -5,13 +5,23 @@
 {% block sideNav %}{% endblock %}
 
 {% block main %}
-  <section class="blurb">{% area article, 'blurb' %}</section>
+  <div class="article-show">
+    <h4 class="meta">{{ data.page.title}}</h4>
+    <h1>{{ data.piece.title }}</h1>
+    <section class="blurb">
+      {% area article, 'blurb' %}
+      <article>{% area article, 'main' %}</article>
```

Comment:
Does it make sense that the article is inside the blurb? I guess that styles well. Just surprised me.

Reply:
I'm removing the blurb from here. It's a thing for the index page.